test/js/testTemplates.js

```diff
@@ -59,10 +66,10 @@ describe('makeNunjucksColors', () => {
         displayColors: ['rgba', 'hsla'],
       },
     });
-    const actual = colors('#fefced');
+    const actual = colors('hsl(53deg 89% 96% / 0.5)');
     const expected = {
-      rgb: 'rgb(254, 252, 237)',
-      hsl: 'hsl(53, 89%, 96%)',
+      rgb: 'rgb(99.56% 98.729% 92.44% / 0.5)',
+      hsl: 'hsl(53 89% 96% / 0.5)',
```

Comment:
@mirisuzanne When users enter `'hsla'` or `'rgba'` as the display format, would you prefer that we display like this, or instead e.g. `hsla(53, 89%, 96% , 0.5)` and `rgba(99.56%, 98.729%, 92.44% , 0.5)`? The former is using the regular rgb/hsl formats with colorjs, the latter is using their rgba/hsla formats.

Reply:
Are we providing separate formats for rgb vs rgba? Their default is the proper modern standard format, so great to prioritize - the latter would only be useful if we're wanting to give people a 'legacy output' option for backwards compatibility.

Reply:
We allow users to enter `'rgba'` or `'hsla'` as display formats, but we currently treat them identically to `'rgb'` and `'hsl'` -- but we could adjust that to use the alternate formats I mentioned above.

Reply:
@mirisuzanne Preference on the above?

Reply:
The advantage is potentially that it lets you get a legacy format - but that wouldn't be entirely clear from simply the input. So maybe I lean towards no change, and we can provide a way to copy-out legacy formats as a distinct feature, maybe as part of support for gamut-mapping.